MachineLearning_Engine/Query/pyCompute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Python.Runtime;
```

Comment:
Copyright header needed. Also this file should probably sit in the `Objects` folder? This isn't a standard BHoM query method and doesn't meet compliance for it?